src/components/DownloadLoginFile.js

```diff
@@ -33,23 +39,23 @@ class DownloadLoginFile extends Nimiq.Observable {
         /** @type {SVGElement} */
         this.$longTouchIndicator = (this.$el.querySelector('.long-touch-indicator'));
 
-        this.$el.addEventListener('mousedown', e => this._onMouseDown(e));
-        this.$el.addEventListener('mouseup', e => this._onMouseUp(e));
+        this.$loginfile.addEventListener('mousedown', e => this._onMouseDown(e));
         this.$loginfile.addEventListener('touchstart', () => this._onTouchStart());
+        this.$downloadButton.addEventListener('click', () => this._onDownloadStart());
         $continueButton.addEventListener('click', this._onDownloadEnd.bind(this));
     }
 
     /**
-     * @param {?HTMLAnchorElement} [$el]
-     * @returns {HTMLAnchorElement}
+     * @param {?HTMLDivElement} [$el]
+     * @returns {HTMLDivElement}
      */
     static _createElement($el) {
-        $el = $el || document.createElement('a');
+        $el = $el || document.createElement('div');
         $el.classList.add('download-loginfile');
 
         /* eslint-disable max-len */
         $el.innerHTML = TemplateTags.noVars`
-            <img class="loginfile" src=""></img>
+            <a class="loginfile-link"><img class="loginfile" src=""></img></a>
```

Comment:
What is that anchor needed for?
I think iOS download should work without the anchor too.

Reply:
The element itself (`this.$el`) is no longer a link, but for iOS long-touch download to work, the image needs to be wrapped in a link with a `href` set. That's what this anchor is for.